### PR TITLE
Add information about synchronous webhook subscription payloads.

### PR DESCRIPTION
**Pull request by jakubkuc**
Add information about synchronous webhook subscription payloads.

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-6294 Handle sync webhooks](https://app.clickup.com/t/2549495/SALEOR-6294)

File: docs/developer/extending/apps/subscription-webhook-payloads.mdx [hunk 1]
```diff
@@ -23,16 +24,17 @@ building a GraphQL query you will be able to use prompts and build the whole sub
 Inside the query, you must define the event type for which the payload will be generated. The event type must be supported by our subscription mechanism (see the [list of supported events](developer/api-reference/unions/event.mdx)).
 
 :::warning
-Only asynchronous webhooks are able to use subscriptions at the moment. Synchronous webhooks will be supported in the future.
+When defining subscription payload for synchronous webhooks some fields are blocked to prevent circular calls.
+More info on which fields are not allowed can be found in Synchro0nous
```
Comment (peelar) [inline docs/developer/extending/apps/subscription-webhook-payloads.mdx:28]: > can be found in Synchro0nous

???

File: docs/developer/extending/apps/subscription-webhook-payloads.mdx [hunk 2]
```diff
@@ -72,28 +72,67 @@ mutation {
...
+    "id": "UGF5bWVudDoyNDE=",
+    "created": "2022-08-03T08:52:04.202449+00:00"
   }
+}
 ```
 
+## Synchronous webhooks payload
+
+Defining synchronous webhooks payloads using subscriptions is also supported. Each event subscription type has access to its specific objects fields.
+Keep in mind that some fields that are using synchronous event to be resolved are not allowed to be called in synchronous webhook payload subscriptions.
```
Comment (peelar) [inline docs/developer/extending/apps/subscription-webhook-payloads.mdx:124]:
```suggestion
Keep in mind that some fields that are using the synchronous event to be resolved are not allowed to be called in synchronous webhook payload subscriptions.
```

File: docs/developer/extending/apps/subscription-webhook-payloads.mdx [hunk 3]
```diff
@@ -72,28 +72,67 @@ mutation {
...
+  "payment": {
+    "id": "UGF5bWVudDoyNDE=",
+    "created": "2022-08-03T08:52:04.202449+00:00"
   }
+}
 ```
 
+## Synchronous webhooks payload
+
+Defining synchronous webhooks payloads using subscriptions is also supported. Each event subscription type has access to its specific objects fields.
```
Comment (peelar) [inline docs/developer/extending/apps/subscription-webhook-payloads.mdx:123]:
```suggestion
Defining synchronous webhooks payloads using subscriptions is also supported. Each event subscription type has access to its specific object fields.
```

File: docs/developer/extending/apps/subscription-webhook-payloads.mdx (hunk 1 above)
Comment (CossackDex) [reply docs/developer/extending/apps/subscription-webhook-payloads.mdx:28]: 🚓

File: docs/developer/extending/apps/subscription-webhook-payloads.mdx (hunk 2 above)
Comment (CossackDex) [inline docs/developer/extending/apps/subscription-webhook-payloads.mdx:124]: Shouldn't this part be in the warning section?

Comment (CossackDex) [reply docs/developer/extending/apps/subscription-webhook-payloads.mdx:124]: Also, shouldn't this sentence be in the warning block?

Comment (CossackDex) [reply docs/developer/extending/apps/subscription-webhook-payloads.mdx:124]:
```suggestion
:::caution
Keep in mind that some fields that are using the synchronous event to be resolved are not allowed to be called in synchronous webhook payload subscriptions.
:::
```

File: docs/developer/extending/apps/subscription-webhook-payloads.mdx (hunk 1 above)
Comment (jakubkuc) [reply docs/developer/extending/apps/subscription-webhook-payloads.mdx]: oops... this was supposed to be deleted. thx.

File: docs/developer/extending/apps/subscription-webhook-payloads.mdx (hunk 2 above)
Comment (jakubkuc) [reply docs/developer/extending/apps/subscription-webhook-payloads.mdx:124]: true

**Review by peelar — APPROVED** [review]

**Pull request merged**